### ISSUE TITLE
[Bug]: Malsorted URLs in Mozilla-manifests

### ISSUE THREAD

**Issue by borque** [russellbanks/Komac#297]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

Hello!
I have provided manifests for the 102.x-series of Mozilla.Firefox.ESR and Mozilla.Thunderbird using yamlcreate.ps1
With the major upgrade to 115.x I wanted to change to Komac.

Unfortunately Komacs current release 1.11.0 completely messed up the installers and URLs while resorting them.
I doublechecked I provided the appropriate URLs requested by Komac, but the result is a completely wrong manifest.

Please see https://github.com/microsoft/winget-pkgs/pull/118488 and https://github.com/microsoft/winget-pkgs/pull/118485 .

Steps to reproduce: use komac update to update the current yamlcreate-sorted manifests.

**Comment by borque** [comment]
This might be related or even a duplicate of https://github.com/russellbanks/Komac/issues/295

**Comment by aaronliu0130** [comment]
I don't think this is related to 295, the specific problem is mismatched installer locales (and maybe installer type). A ton of different locales were labeled de-DE.

**Comment by aaronliu0130** [comment]
Note that while I was updating the betterbird manifest today, Komac did not switch around the order. https://github.com/microsoft/winget-pkgs/pull/119776/files
